unstructured_ingest/v2/processes/connectors/kafka.py

```diff
@@ -0,0 +1,324 @@
+import base64
+import json
+import socket
+import typing as t
```

Comment:
Can we use the explicit import rather than an alias? 
```suggestion
from typing import Optional, ...
```